packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/exceptions/PangeaAPIException.java

```diff
@@ -14,4 +15,17 @@ public PangeaAPIException(String message, Response<PangeaErrors> response){
     public Response<PangeaErrors> getResponse() {
         return response;
     }
+
+    public String toString(){
```

Comment:
nit: to build a String you should use StringBuilder instead of concatenating the pieces.